libr/util/assert.c

```diff
@@ -7,7 +7,7 @@
 R_API void r_assert_log(RLogLevel level, const char *fmt, ...) {
 	va_list args;
 	va_start (args, fmt);
-	r_vlogf (level, fmt, args);
+	R_LOG (level, NULL, fmt, args);
```

Comment:
this is wrong because:
1) it uses NULL where there should be `fmt` (which makes the function segfault)
2) _r_log_internal called by R_LOG does not accept a va_list but variadic arguments.

The thing should be refactored a bit so to provide a R_VLOG version as it was before, that accepts a va_list instead of multiple arguments, and `r_assert_log` should call the R_VLOG instead of R_LOG. But... We've already asked many things to you in this PR and I'm willing to clean this up myself when merging if you don't mind. :)

Reply:
I changed R_LOG back to `R_LOG(lvl, tag, fmtstr, ...)` so you can specify a custom tag again with it. The other macros just pass NULL so they are still `(fmtstr, ...)`.

Also I guess I need to read up on how the varadic stuff works in C, I thought the `...` was some sort of inferred va_list.